ivy_tests/test_ivy/test_frontends/test_tensorflow/test_sets.py

```diff
@@ -0,0 +1,38 @@
+# global
+import hypothesis.strategies as st
+import tensorflow as tf
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.test_functional.test_core.test_statistical import (
+    _statistical_dtype_values,
+)
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+@handle_frontend_test(
+    fn_tree="tf.sets.intersection",
```

Comment:
You need to spell out the framework name as it is in the folder structure (i.e. `tensorflow.sets.intersection`)